udf/bits_to_last_seen_int.sql

```diff
@@ -0,0 +1,41 @@
+/*
+Given a BYTE, return the number of days since the client was
```

Comment:
```suggestion
Given a single-entry BYTES array, return the number of days since the client was
```

Is the above what you mean by a BYTE?

Reply:
No, I do mean `BYTES` though!

Reply:
I'm specifically leaving out the word `array` since that may confused people, since arrays are a different data structure.